themes/bootstrap3/templates/cart/cart.phtml

```diff
@@ -69,7 +69,7 @@
       submitFormWithButton(this, 'empty');
   });
   $("#cart-refuse-empty").click(function($e) {
-      $('.fa.fa-spinner').remove();
+      $('.loading-spinner').remove();
```

Comment:
I couldn't figure out what circumstances required this code to exist, so I wasn't sure how to test it. Do you remember?

Reply:
It looks like rejecting the confirmation to empty the cart.

Reply:
That's correct... but bringing up the confirmation doesn't add a loading icon, so I'm not sure what its purpose is. (I did confirm that the code triggers, I just couldn't figure out when it would trigger with a loading icon on the screen).